MekHQ/src/mekhq/gui/dialog/ManageAssetsDialog.java

```diff
@@ -161,18 +139,22 @@ private void assetTableValueChanged(javax.swing.event.ListSelectionEvent evt) {
     private void addAsset() {
         Asset a = new Asset();
         EditAssetDialog ead = new EditAssetDialog(frame, a);
+        ead.setTitle(resourceMap.getString("addAssetDialogTitle.text"));
         ead.setVisible(true);
         if(!ead.wasCancelled()) {
+            ead.dispose();
```

Comment:
if this is cancelled should it still be disposed?

Reply:
Yep, good catch. I will do some more work on these dialogs after this PR.